sqlite/src/status.c

```diff
@@ -351,7 +364,7 @@ int sqlite3_db_status(
     */
     case SQLITE_DBSTATUS_CACHE_SPILL:
       op = SQLITE_DBSTATUS_CACHE_WRITE+1;
-      /* Fall through into the next case */
+      /* no break */ deliberate_fall_through
```

Comment:
how does this compile?

Reply:
The "deliberate_fall_through" thing is a macro that either expands to nothing or to a special GCC construct that disables the compiler warning.